themes/kf/sass/src/index.html

```diff
@@ -159,7 +159,7 @@ <h4>TensorFlow model training</h4>
 				</div>
 				<div class="text">
 					<h4>Model serving</h4>
-					<p>A TensorFlow Serving container to export trained TensorFlow models to Kubernetes. We also integrate with Seldon Core, the open source platform for deploying machine learning models on Kubernetes.</p>
+					<p>A TensorFlow Serving container to export trained TensorFlow models to Kubernetes. We also integrate with Seldon Core, an open source platform for deploying machine learning models on Kubernetes, and NVIDIA TensorRT Inference Server for maximized GPU utilization when deploying ML/DL models at scale.</p>
```

Comment:
This file shouldn't need any changes. The only things pulled out of sass directories are the compiled style changes.